andela/bp-esa-frontend
# [Feature #160437219(Sub-Task) Implement search with partner name or fellow name

**What does this PR do?**
Add the search component for the report page
Add searches to the constants.js file.
Refactor the mockReport.js to reflect the different partner and fellow names.

**Description of Task to be completed?**
User should be able to search through automation reports with partner or fellow name.

**How should this be manually tested?**

Clone and set up the project using the instruction in the README.
Use $ git checkout ft-filter-report-with-fellow-name to check out the branch containing the changes of this pull request
Use $ npm start to start the application
Visit http://localhost:3000/ to see the report page.
You should see the search input type any of the partner or fellow names. 
Click on the drop down and have a filtered report.
Remove the search option and all the data will reappear.
**What are the relevant pivotal tracker stories?**
[#160437219](https://www.pivotaltracker.com/story/show/160437219)

Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-10-30 at 11 52 40" src="https://user-images.githubusercontent.com/6957960/47706551-8f963580-dc3a-11e8-93bd-0de27984e994.png">
<img width="1440" alt="screen shot 2018-10-30 at 11 53 09" src="https://user-images.githubusercontent.com/6957960/47706556-93c25300-dc3a-11e8-907e-ae4b717f1ffc.png">
